test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml

```diff
@@ -11,9 +11,16 @@ spec:
       labels:
         app: gcp-compute-persistent-disk-csi-driver
     spec:
+      # Host network must be used for interaction with Workload Identity in GKE
+      # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
+      # this requirement when issue is resolved and before any exposure of
+      # metrics ports.
+      hostNetwork: true
+      priorityClassName: csi-gce-pd-node
```

Comment:
I don't think these two are defined here. Also node shouldn't need a service account since it doesn't make K8s calls.

Reply:
ah how did those sneak in